src/genie/libs/parser/dell/os10/tests/ShowIPInterfaceBrief/equal/golden_output_expected.py

```diff
@@ -0,0 +1,242 @@
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/20': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/21': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/22': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/23': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/24': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/25': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/26': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/27': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/28': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/29': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/30': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/31': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Ethernet 1/1/32': {
+            'ip_address': 'unassigned',
+            'ok': False,
+            'method': 'unset',
+            'status': 'up',
+            'protocol': 'down'
+        },
+        'Management 1/1/1': {
+            'ip_address': '10.10.21.16/24',
+            'ok': True,
+            'method': 'manual',
+            'status': 'up',
+            'protocol': 'up'
+        },
+        'Vlan 1': {
+            'ip_address': 'unassigned',
+            'ok': True,
```

Comment:
below is also OK. no need to change. but recently, trying to parse the value as is as much as possible. so that we can easily compare between parsed_output and actual output.
```
'ok': 'YES',